src/main/java/persistence/sql/ddl/query/ColumnQueryBuilder.java

```diff
@@ -8,36 +8,27 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Optional;
 import java.util.stream.Collectors;
 
 public class ColumnQueryBuilder {
 
-    public final List<Constraint> constraints = new ArrayList<>();
+    private final List<Constraint> constraints;
 
-    public ColumnQueryBuilder() {
-        constraints.add(new NotNullConstraint());
-        constraints.add(new IdentityConstraint());
-        constraints.add(new PrimaryKeyConstraint());
+    public ColumnQueryBuilder(List<Constraint> list) {
+        this.constraints = list;
     }
```

Comment:
생성자 주입 피드백 반영 👍  변수명도 함께 의미 있는 이름으로 지어주면 좋을 것 같아요 :)

```suggestion
    public ColumnQueryBuilder(List<Constraint> constraints) {
        this.constraints = constraints;
    }
```